OpenLIFULib/OpenLIFULib/solution.py

```diff
@@ -53,6 +53,10 @@ def initialize_from_openlifu_data(
         pnp_volume_node.GetDisplayNode().SetAndObserveColorNodeID("vtkMRMLColorTableNodeFilePlasma.txt")
         intensity_volume_node.GetDisplayNode().SetAndObserveColorNodeID("vtkMRMLColorTableNodeFilePlasma.txt")
 
+        # Set openlifu solution attribute
+        pnp_volume_node.SetAttribute('isOpenLIFUSolution', 'True')
+        intensity_volume_node.SetAttribute('isOpenLIFUSolution', 'True')
+
```

Comment:
This is great! Can we also use these tags to omit the volume from the list of loaded openlifu objects displayed in the openlifu data module? This could also be considered part of #115.

Reply:
My opinion:

I think they should be omitted because they are kind of built into the "Solution" which is displayed already as a loaded openlifu object

As slicer nodes they are their own thing independent of the Solution, sure

But as far as the openlifu world is concerned they are just there as a way to display something that is really in the Solution

Similar to the situation with trasnducer transform and model node -- they are two slicer nodes but in the openlifu world they are considered part of the one openlifu transducer